src/lesson/lesson03.js

```diff
@@ -518,46 +522,852 @@ Lesson03ExerciseStepPlayer.prototype = {
 function Lesson03() {
   Lesson.Lesson.call(this);
 
+  // Step 1
   this.addStep(
     new Lesson.LessonStep(
       <p>
-        Comandos: P (put), G (get), F (forward), L (left), R (right).
-        Sensores: eng (engrenagem), maq (máquina). Exemplo: {"eng?{G}"}
+        O robô precisa andar para frente e pegar a engrenagem.
       </p>,
       <div>
         <p>
-          Objetivo: pegar engrenagem.
+          Olá! Da última vez que nos encontramos conseguimos coletar muitas
+          baterias, e agora temos bastante energia para que o robô consiga
+          cumprir a sua missão. Agora, precisamos garantir que todas as
+          máquinas na nossa base de operações estejam funcionando.
+        </p>
+        <p>
+          A chuva de
+          meteoros provocou tremores na nossa base e várias máquinas pararam de
+          funcionar, porque suas engrenagens se soltaram. Elas não
+          estavam preparadas para tanto impacto. A missão do robô agora
+          é pegar todas as engrenagens e
+          colocá-las de volta nas máquinas.
+        </p>
+        <p>
+          Olhe só, nesse cenário existe uma engrenagem solta.
+          Você precisa fazer o robô pegá-la. Para isso, você só precisa
+          utilizar o comando “<b>F</b>” (do inglês <i>Forward</i>,
+          que significa <b>pra frente</b>) para o robô andar uma vez
+          e ficar em frente à engrenagem, e depois utilizar
+          o comando “<b>G</b>” (do inglês <i>Get</i>, que
+          significa <b>pegar</b>) para o robô pegar a engrenagem.
+        </p>
+      </div>,
+      new Lesson03ExerciseStepPlayer(
+        3, // rows
+        3, // cols
+        new Grid.Position(1, 0), // robot_position
+        Grid.Directions.RIGHT, // robot_direction
+        false, // initially_holding
+        [new Grid.Position(1, 2)], // components_positions
+        [], // machines_positions
+        // [], // hidden_positions
+        Goals.GET_ALL_COMPONENTS // goal
+      ),
+      "",  // initialCode
+      Constants.Lesson03.SUCCESS_MESSAGE,
+      null
+    )
+  );
+
+  // Step 2
+  this.addStep(
+    new Lesson.LessonStep(
+      <p>
+        O robô precisa chegar em frente à máquina e colocar a engrenagem.
+      </p>,
+      <div>
+        <p>
+          Muito bem, o robô pegou uma engrenagem, e está segurando
+          ela. Agora, ele precisa colocá-la em uma máquina que
+          está com uma engrenagem faltando.
+          Para isso, ele precisa chegar em frente à máquina e utilizar o seu
+          comando “<b>P</b>” (do inglês <i>Put</i>, que significa
+          <b>colocar</b>).
```

Comment:
By default, there's no spacing around tags when they start in a new line, so the text reads "significacolocar" (with the "colocar" in bold). You can put an explicit space: add &nbsp<b></b>; before <b><</b>b>colocar<b><</b>/b>